File: fix-faq6-ru.js
const fs = require('fs');

const content = fs.readFileSync('/src/locales/translations.ts', 'utf8');

// Replace single \n after Russian FAQ6 headings with double \n\n
let fixed = content;
fixed = fixed.replace('1. Юридическая защита (US Law)\\nКомпания', '1. Юридическая защита (US Law)\\n\\nКомпания');
fixed = fixed.replace('2. Интеллектуальная собственность (IP Ownership)\\nВсе', '2. Интеллектуальная собственность (IP Ownership)\\n\\nВсе');
fixed = fixed.replace('3. Прозрачность и отчетность (Reporting)\\nИнвесторы', '3. Прозрачность и отчетность (Reporting)\\n\\nИнвесторы');
fixed = fixed.replace('4. Мотивация основателей (Vesting)\\nАкции', '4. Мотивация основателей (Vesting)\\n\\nАкции');

fs.writeFileSync('/src/locales/translations.ts', fixed, 'utf8');
console.log('Fixed Russian FAQ6 formatting!');

File: fix_faq6_ru.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

with open('/src/locales/translations.ts', 'r', encoding='utf-8') as f:
    content = f.read()

# Replace single \n after Russian FAQ6 headings with double \n\n
replacements = [
    ('1. Юридическая защита (US Law)\\nКомпания', '1. Юридическая защита (US Law)\\n\\nКомпания'),
    ('2. Интеллектуальная собственность (IP Ownership)\\nВсе', '2. Интеллектуальная собственность (IP Ownership)\\n\\nВсе'),
    ('3. Прозрачность и отчетность (Reporting)\\nИнвесторы', '3. Прозрачность и отчетность (Reporting)\\n\\nИнвесторы'),
    ('4. Мотивация основателей (Vesting)\\nАкции', '4. Мотивация основателей (Vesting)\\n\\nАкции'),
]

for old, new in replacements:
    content = content.replace(old, new)

with open('/src/locales/translations.ts', 'w', encoding='utf-8') as f:
    f.write(content)

print("Fixed Russian FAQ6 formatting!")
